build/Defaults/app.config

```diff
@@ -19,7 +18,51 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
-        <bindingRedirect oldVersion="0.0.0.0-1.2.0.0" newVersion="1.2.0.0"/>
+        <bindingRedirect oldVersion="0.0.0.0-1.2.1.0" newVersion="1.2.1.0"/>
+      </dependentAssembly>
```

Comment:
For example, System.Reflection.Metadata assembly reference is to System.Collections.Immutable 1.2.0 (it presumably doesn't need newer version), however the contract assembly in package System.Collections.Immutable package has version 1.2.1.